app/posts/ecf-v2/initial-release-of-ecf-2.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Deciding what to build for the initial release of ECF 2
+description: "What we want to build to replace and improve the digital services that facilitate the early career framework (ECF) policy reforms"
+date: 2024-07-11
+---
+
+## What we've been working on
+
+Over the last two months, we have been working on a discovery to explore the biggest problems in the existing digital services that facilitate the early career framework (ECF) policy reforms.
+
+As part of this work, we had to define the minimum viable product for what we need to build to replace them. The minimum viable product, or MVP, is what we need to build for the new services, before we can initially release them. In this context, it's the minimum we need to make the services usable and fulfill their core purposes.
+
+We need a MVP because it's impossible to address all the problems or build all the solutions at once. It's better to release something early that users can test, which we can then improve. The smaller this initial release, or MVP, is, the sooner we can do this.
+
+Other new features and iterations will come to these services in the future.
+
+The services we considered in our discovery included:
+* Check data for appropriate bodies
+* Check data for delivery partners
+* Manage training for early career teachers
+* The lead provider API and its documentation
+* Internal finance and support tooling for DfE colleagues
+
+Another design history will detail the problems we discovered and potential ideas to address them.
+
+Here, we'll define what we currently think we need to build for the initial release of the new digital services for ECF.
+
+## How we decided what we would include in the MVP
+
+We decided what we needed to include for the MVP for the new digital services by considering:
+* what the current ECF services do
+* the core purposes of the ECF digital services
+* our ideas for improvements to these services based on the problems we discovered
+* how we could increase flexibility, self-sustainability and autonomy for these services
+
+## The MVP for the new digital services for ECF 
+
+We've aligned everything we want to do or build for MVP to the core purposes for [Teacher Continuing Professional Development programme](https://teacher-cpd.design-history.education.gov.uk/cross-programme/our-refreshed-vision/).
+
+Here is a map showing what we hope to build for the MVP of the new digital services.
+
+![A map showing what we want to build for the MVP of the new digital services. It includes building a new Lead Provider API, a database just for ECF with an improved model, a service for appropriate bodies to check training data, joint support and finance tooling for internal users and a service for schools to register early career teachers and mentors.](app/images/ecf-2/initial-release-of-ecf-2/mvp-map.png)
```

Comment:
Image doesn't load for me when running in codespace, I think can be fixed by changing it to
`(/ecf-2/initial-release-of-ecf-2/mvp-map.png)`